.gitignore

```diff
@@ -96,7 +96,10 @@ doc/preconditioners/lgr_navier_stokes/lgr_navier_stokes_old2.txt
 *.swo
 
 # User drivers should go in your own version control I guess
-user_drivers/
+user_drivers/jack_cool
+user_drivers/joe_cool
+
+demo_drivers/meshing/mesh_from_gmsh/RESLT/
```

Comment:
This directory should never exist. All tests should be run in the Validation directory which is deleted by make clean.